style.css

```diff
@@ -191,16 +203,110 @@ nav .buttons {
     overflow: auto;
 }
 
-.coding_langs img {
-    float:left;
-}
+/*.coding_langs img {*/
+    /*float:left;*/
+/*}*/
 
 main .body_container {
     padding-top: 30px;
 }
 
+
+
 #portfolioId {
-   background-color: #4b3341;
+    background-color: #9bbab0;
+    height: 1000px;
+    padding-top: 100px;
+    padding-right: 135px;
+    padding-left: 135px;
+    padding-bottom: 100px;
+
+}
+
+.portfolio_title {
+    font-family: 'Stalemate', cursive;
+    font-size: 5rem;
+    width: 200px;
+    margin: 0 auto;
+}
+
+.portfolio_container {
+    overflow: auto;
+    width: 800px;
+    margin: 0 auto;
+}
+
+.portfolio_button {
+    height: 250px;
+    width: 250px;
+    border: #4b3341 solid 2px;
+    text-align: center;
+    display: inline-block;
+    border-radius: 10px;
+}
+
+.portfolio_button:hover {
+    background-color: #4b3341;
+}
+
+@media screen and (max-width: 1080px) {
+    .portfolio_container {
+        width: 650px;
+    }
+
+    .portfolio_button {
+        height: 200px;
+        width: 200px;
+    }
+}
+
+@media screen and (max-width: 935px) {
+    .portfolio_title {
+        font-size: 5rem;
+        width: 200px;
+        margin: 0 auto;
+    }
+
+    .portfolio_container {
+        width: 520px;
+    }
+
+    .portfolio_button {
+        height: 250px;
+        width: 250px;
+    }
+}
+@media screen and (max-width: 800px) {
+    .portfolio_button {
+        height: 175px;
+        width: 100%;
+    }
+    .portfolio_container {
+        width: 400px;
+    }
+
+    #portfolioId {
+
+        padding-right: 10%;
+        padding-left: 10%;
+
+    }
+}
+@media screen and (max-width: 800px) {
+    .portfolio_button {
+        height: 150px;
+        width: 100%;
+    }
+    .portfolio_container {
```

Comment:
you might want to change the width of your container - when your minimize the size of the page eventually the right bar disappears